llvm/include/llvm/TargetParser/X86TargetParser.def

```diff
@@ -241,6 +241,7 @@ X86_FEATURE       (SM3,             "sm3")
 X86_FEATURE       (SM4,             "sm4")
 X86_FEATURE       (AVXVNNIINT16,    "avxvnniint16")
 X86_FEATURE       (EVEX512,         "evex512")
+X86_FEATURE       (USERMSR,            "usermsr")
```

Comment:
Make it aligned above

Reply:
3e32e495d8fa56ad3770769a5d4559ea0a41af96